### PR TITLE
LibWeb: Update DOM IDL specs

### PR DESCRIPTION
**Pull request by gmta**
I noticed some of these were running behind the upstream spec.

### CODE REVIEW

File: Userland/Libraries/LibWeb/DOM/Event.idl [hunk 1]
```diff
@@ -19,21 +19,20 @@ interface Event {
...
     undefined preventDefault();
     readonly attribute boolean defaultPrevented;
     readonly attribute boolean composed;
 
-    readonly attribute boolean isTrusted;
-    readonly attribute double timeStamp;
-
-    undefined initEvent(DOMString type, optional boolean bubbles = false, optional boolean cancelable = false);
+    [LegacyUnforgeable] readonly attribute boolean isTrusted;
+    readonly attribute DOMHighResTimeStamp timeStamp;
```
Comment (tcl3) [inline Userland/Libraries/LibWeb/DOM/Event.idl:33]: `Event::time_stamp()` should probably be updated to return `DOMHighResTimeStamp` as well, for clarity.

Comment (gmta) [reply Userland/Libraries/LibWeb/DOM/Event.idl:33]: Absolutely!

File: Userland/Libraries/LibWeb/DOM/StaticRange.idl [hunk 2]
```diff
@@ -2,14 +2,14 @@
 #import <DOM/AbstractRange.idl>
 
 // https://dom.spec.whatwg.org/#staticrange
```
Comment (trflynn89) [inline Userland/Libraries/LibWeb/DOM/StaticRange.idl:4]: The spec links are a bit off here now - this one belongs over `StaticRange`, and and `https://dom.spec.whatwg.org/#dictdef-staticrangeinit` goes over `StaticRangeInit` (although we didn't have the latter at all before)

Comment (gmta) [reply Userland/Libraries/LibWeb/DOM/StaticRange.idl]: Good catch - did not know that individual definitions had their own permalinks, which makes sense. We're probably missing those in a lot of places.

**Pull request merged**